packages/framework/store/src/__tests__/block.unit.spec.ts

```diff
@@ -55,3 +57,142 @@ test('init block without props should add default props', () => {
   expect(model.count).toBe(0);
   expect(model.style).toEqual({});
 });
+
+describe('block model should has signal props', () => {
+  test('atom', () => {
+    const doc = createTestDoc();
+    const yDoc = new Y.Doc();
+    const yBlock = yDoc.getMap('yBlock') as YBlock;
+    yBlock.set('sys:id', '0');
+    yBlock.set('sys:flavour', 'page');
+    yBlock.set('sys:children', new Y.Array());
+
+    const block = new Block(doc.schema, yBlock, doc);
+    const model = block.model as RootModel;
+
+    const isOdd = computed(() => model.count$.value % 2 === 1);
+
+    expect(model.count$.value).toBe(0);
+    expect(isOdd.peek()).toBe(false);
```

Comment:
Just a question, why do we need `peek` here.

Reply:
We don't. I just test all the functionalities.